test/contracts/TalentFactory.ts

```diff
@@ -30,6 +30,7 @@ describe("TalentFactory", () => {
     [creator, minter, talent1, talent2] = await ethers.getSigners();
 
     TalentFactoryFactory = await ethers.getContractFactory("TalentFactory");
+    upgrades.silenceWarnings()
```

Comment:
should we revisit this? cc/ @verdefred

Reply:
yes, I'll try to spend some time to figure out more in depth why this was needed and try to remove it